tools/gn

```diff
@@ -617,11 +617,8 @@ def to_gn_wasm_args(args, gn_args):
   gn_args['skia_use_vulkan'] = False
   gn_args['skia_use_webgpu'] = False
   gn_args['skia_use_libheif'] = False
-  gn_args['skia_use_libjpeg_turbo_decode'] = True
   gn_args['skia_use_libjpeg_turbo_encode'] = False
-  gn_args['skia_use_libpng_decode'] = True
   gn_args['skia_use_libpng_encode'] = True
```

Comment:
Can you please file an issue to remove this as well? We can implement this by piping the image through a 2D canvas.